### DISCUSSION TITLE
v0.1.9

### DISCUSSION THREAD

**Discussion by cyring** [Umio-Yasuno/amdgpu_top#20] [Show and tell]
New `v0.1.9` does not fit 80 columns as the previous version did; especially in `fdinfo` and `Sensors`

![2023-06-04-152703_644x942_scrot](https://github.com/Umio-Yasuno/amdgpu_top/assets/11563789/15985650-919b-48a1-b4ae-35d7792a61a1)

It now requires 90 characters:

![2023-06-04-152750_724x872_scrot](https://github.com/Umio-Yasuno/amdgpu_top/assets/11563789/bbddee17-70de-4aea-a954-90a4ece1d3fa)

Could the UI be optimized for the bare standard Terminal width ?

**Comment by Umio-Yasuno** [comment]
https://github.com/Umio-Yasuno/amdgpu_top/commit/816d314b97bcb2b5df164074619ac1da7f8315cd

**Reply by cyring** [reply]
Refreshed from latest 46516700ac786c6a98b9d173d0caa39afedd51a2

* In a `80x66` XTerm:

![2023-06-09-201320_644x928_scrot](https://github.com/Umio-Yasuno/amdgpu_top/assets/11563789/7d2b64dc-0779-4d15-9d3a-8745d4525531)

* `80x24` to force scrolling:

![2023-06-09-202040_644x340_scrot](https://github.com/Umio-Yasuno/amdgpu_top/assets/11563789/68beece9-af36-4fae-b9ee-443cad837768)

* Simple TUI
 1. `80x24`

![2023-06-09-202338_644x340_scrot](https://github.com/Umio-Yasuno/amdgpu_top/assets/11563789/dd983fd3-7875-4a2a-94f6-fca818a17f87)

 2. `99x24`

![2023-06-09-202625_796x340_scrot](https://github.com/Umio-Yasuno/amdgpu_top/assets/11563789/a1714e86-46e7-4a53-96f6-55bef3543776)

**Reply by Umio-Yasuno** [reply]
smi: adjust to 80 columns
https://github.com/Umio-Yasuno/amdgpu_top/commit/e61e2961a48563224255f1100c7a0cd8136b1d97
  
If you have any other issues with v0.1.9, please create an issue.

**Reply by cyring** [reply]
> smi: adjust to 80 columns [e61e296](https://github.com/Umio-Yasuno/amdgpu_top/commit/e61e2961a48563224255f1100c7a0cd8136b1d97)
> 
> If you have any other issues with v0.1.9, please create an issue.

It is adjusted, thanks
